app/controllers/downloads_controller.rb

```diff
@@ -25,19 +25,34 @@ def gui
   def download
     @platform = params[:platform]
     @platform = 'windows' if @platform == 'win'
-    if @platform == 'windows' || @platform == 'mac'
-      if @platform == 'windows'
-        @project_url  = "https://msysgit.github.io/"
-        @source_url   = "https://github.com/msysgit/git/"
-      else
-        @project_url = "http://sourceforge.net/projects/git-osx-installer/"
-        @source_url   = "https://github.com/git/git/"
-      end
+    if @platform == 'mac'
+      @project_url = "http://sourceforge.net/projects/git-osx-installer/"
+      @source_url   = "https://github.com/git/git/"
 
       @download = Download.latest_for(@platform)
       @latest = Version.latest_version
 
       render "downloads/downloading"
+    elsif @platform == 'windows'
+      @project_url = "https://git-for-windows.github.io/"
+      @source_url = "https://github.com/git-for-windows/git"
+
+      @download32 = Download.latest_for(@platform + "32")
+      @download64 = Download.latest_for(@platform + "64")
+      @download32portable = Download.latest_for(@platform + "32Portable")
+      @download64portable = Download.latest_for(@platform + "64Portable")
+      @latest = Version.latest_version
+
+      agent = request.env["HTTP_USER_AGENT"]
+      if agent.include? "WOW64" || agent.include? "Win64"
+        @download = @download64
+        @bitness = "64bit"
```

Comment:
Since these are intended for output to the user, perhaps they should read "64-bit" and "32-bit" instead of "64bit" and "32bit".